File: OpenWeatherMap.Data/Models/City/City.cs
using System.Runtime.Serialization;

namespace OpenWeatherMap.Data.Models.City
{
    [DataContract]
    public class City
    {
        [DataMember]
        public int id { get; set; }
        [DataMember]
        public string name { get; set; }
        [DataMember]
        public string country { get; set; }
        [DataMember]
        public Coord coord { get; set; }
    }
}

File: CotecnaTest/ClientApp/src/app/models/City.ts
import { Coord } from "./Coord";


export class City {
  id: number;
  name: string;
  country: string;
  coord: Coord;

  constructor(id: number, name: string, country: string, coord: Coord) {
    this.id = id;
    this.name = name;
    this.country = country;
    this.coord = coord;
  }

}
